src/utils.rs

```diff
@@ -615,6 +616,47 @@ pub(crate) fn unicode_str_width(s: &str) -> usize {
     s.width()
 }
 
+#[cfg(windows)]
+pub fn absolute_path<P: AsRef<path::Path>>(p: P) -> io::Result<path::PathBuf> {
+    use std::ffi::OsString;
+    use std::iter::once;
+    use std::os::windows::ffi::{OsStrExt, OsStringExt};
+    use std::ptr::null_mut;
+    use winapi::um::errhandlingapi::GetLastError;
+    use winapi::um::fileapi::GetFullPathNameW;
+
+    // FIXME: This `MAX_PATH` may be valid only from Windows 10, version 1607.
+    // https://docs.microsoft.com/ja-jp/windows/desktop/FileIO/naming-a-file#paths
+    const MAX_PATH: usize = 32767;
```

Comment:
It seems like you cannot use `\\?\` with relative path according to https://docs.microsoft.com/en-gb/windows/desktop/FileIO/naming-a-file#maximum-path-length-limitation

>  Because you cannot use the "\\?\" prefix with a relative path, relative paths are always limited to a total of MAX_PATH characters.

Reply:
I tried to prefix the path with `\\?\` and I get an error:

```
The specified path is invalid. (os error 161)
```

Reply:
Must be in *absolute* path form.